selfdrive/car/ford/fordcan.py

```diff
@@ -128,17 +128,17 @@ def create_acc_msg(packer, CAN: CanBus, long_active: bool, gas: float, accel: fl
   Frequency is 50Hz.
   """
 
-  decel = accel < 0 and long_active
+  actuateBrakes = gas == -5 and long_active
```

Comment:
```suggestion
  actuate_brake = gas < CarControllerParams.MIN_GAS and long_active
```

magic number

Reply:
Don't we want the `actuate_brake` signal to occur only when our gas is at `CarControllerParams.MIN_GAS`? 

Also, I could be reading this wrong but 
```
if not CC.longActive or gas < CarControllerParams.MIN_GAS:
        gas = CarControllerParams.INACTIVE_GAS
```

wouldn't `actuate_brake` never fire because we're always setting gas to -5 before we get here? 

https://github.com/coffee-cake-studios/openpilot/blob/a7692bffd14e97a879da5359a0939f84e88efed6/selfdrive/car/ford/carcontroller.py#L90

Reply:
Ah right, then you can check accel

Reply:
Ah right, then you can check accel

Reply:
Ah right, then you can check accel

Reply:
Let me make sure I understand `actuate_brake = gas < CarControllerParams.ACCEL_MIN and long_active`? This could work well, too. Start braking if the model wants to accelerate at less than 3.5m/s^2?

Reply:
No, gas is a copy of accel, minus setting to inactive gas. This check here matches what carcontroller does.

Reply:
No, gas is a copy of accel, minus setting to inactive gas. This check here matches what carcontroller does.

Reply:
Okay, I see lmfao. My bad, I was reading it wrong. 

`actuate_brake = accel < CarControllerParams.MIN_GAS and long_active`